__tests__/app.test.js

```diff
@@ -147,5 +147,67 @@ describe('GET /api/articloos', () => {
 
 
 
+describe("6.GET /api/articles/:article_id/comments", () => {
+  test("status 200: repondes with an array of comments for a given artical_id", () => {
+
+    return request(app)
+      .get(`/api/articles/1/comments`)
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments.length).toBe(11);
+        comments.forEach((comment) => {
+          expect(comment).toMatchObject(
+            expect.objectContaining({
+              comment_id: expect.any(Number),
+              author: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              body: expect.any(String),
+            })
+          );
+        });
+      });
+  });
+  test('status 200, respondes with sorted array according to given query', () => { 
+    return request(app)
+      .get("/api/articles/1/comments?sort_by=created_at")
+      .expect(200)
+      .then(({ body: { comments } }) => {
+        expect(comments).toBeSortedBy('created_at', { descending: true });
+      });
+    
+  })
+  test("status 200, expect response to be an array of  sorted by given query", () => {
+    return request(app)
+      .get("/api/articles/1/comments?sort_by=votes")
+      .expect(200)
+      .then(({ body: { comments } }) => {
+        expect(comments).toBeSortedBy("votes", { descending: true });
+      });
+  });
+});
```

Comment:
Please add another test where there are no comments for an existing article. This should expect status 200 and an empty array.